### ISSUE TITLE
UX improvement

### ISSUE THREAD

**Issue by nimishbongale** [creativecommons/vue-vocabulary#224]
**Description**

1. It so happens that the top page tabs ("Canvas and Docs") make sense only when we click on a component from the left vertical bar. Otherwise clicking on them does nothing, apart from a dummy switch between the two.

2. The top menu bar and the left vertical column get highlighted with a thin black background on clicking on it. 

**Reproduction**

Just running `npm run stoybook` or through docker and visiting localhost is sufficient to notice the issue.

**Expected behavior**

1. The upper menu bar options should be enabled/made visible only after selecting a component from the left vertical menu.
2, Ideally, nothing should happen.

**Screenshots**

![Screenshot (1)](https://user-images.githubusercontent.com/43414361/83338254-2841db00-a2e0-11ea-804b-83245fa9d8c4.png)

**Want to take up?**
Yes!

**Comment by dhruvkb** [comment]
@nimishbongale it seems to me that you have discovered a bug (or lack of a feature) with the tool in question, i.e. Storybook and not with Vocabulary. It seems to have a fix in an upcoming Storybook version [6.0.0-alpha.2](https://github.com/storybookjs/storybook/releases/tag/v6.0.0-alpha.2). We'll just have to wait for it to be released.

**Comment by nimishbongale** [comment]
@dhruvkb Ahh alright. 

Maybe, instead of the manual wait, we can configure a "dependabot" (owned by github) to do the needful? It creates an automated pull request to bump up dependencies ( only stable versions ).

* https://dependabot.com/

As of now, the latest released `storybook.js` version is 5.3.19. I was working with 5.3.18 from the looks of it,

Let me know if this `dependabot` configuration can be done, and I'll be willing to set it up for this repo. 

Thanks!

**Comment by nimishbongale** [comment]
@dhruvkb @kgodey 

Github released some major improvements/features to dependabot. Just fyi.

* https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/

![depend](https://user-images.githubusercontent.com/43414361/83915200-6af82d00-a790-11ea-8e8e-51bb7d8fe034.PNG)

Thanks!

**Comment by dhruvkb** [comment]
@nimishbongale I've separated the package-update aspect of this into a separate issue at #230.

**Comment by nimishbongale** [comment]
@dhruvkb Alright! 

Maybe this issue can remain open until the new `storybook.js` is relased.

**Comment by kgodey** [comment]
Closing this since it's an issue with Storybook.
